code/game/gamemodes/changeling/helpers/_store.dm

```diff
@@ -53,10 +53,17 @@ var/list/datum/power/changeling/powerinstances = list()
 /datum/power/changeling/fakedeath
 	name = "Regenerative Stasis"
 	desc = "We fake our own death, imperceptible to even the best of doctors. We can choose when to re-animate."
-	helptext = "Can be used before or after death. Duration varies greatly."
+	helptext = "Can only be used before death. CAN ONLY BE USED ONCE."
 	genomecost = 0
 	verbpath = /mob/proc/changeling_fakedeath
 
+/datum/power/changeling/emergency_transform
+	name = "Emergency Transform"
+	desc = "If circumstances are dire enough, we can detach our head to transform into a lesser form to escape and regenerate later."
+	helptext = "Can be used before or after death."
```

Comment:
if it can be used after death, doesn't this bypass the single reg. stasis?
will this not encourage lings to just decouple their head and ventcrawl then AFK somewhere for 10 minutes each time they die?

Reply:
it does bypass the single reg stasis yeah, they can pop their heads off infinitely. but they can only regen stasis once since it's tracked on the datum that gets passed to the new mob

sure, maybe they AFK somewhere for 10 minutes, but that's 10 minutes the crew can search for them, heal up, or get better weapons and armor. next time it happens, the crew can put them somewhere they can't ventcrawl.